src/content/experimental/ui-shell/usage.md

```diff
@@ -30,22 +29,41 @@ The **Local** zone is controlled at the product level. It contains the product-l
 
 ## UI Shell Components
 
+The UI Shell is designed to be configurable. A product/platform can choose which shell components and configurations to use.
+
 ### Primary Header
 
-The Header spans the full width of the viewport and is the topmost element in the UI. Header elements are persistent within a product. The header contains:
+The Header spans the full width of the viewport and is the topmost element in the UI. Header elements are persistent within a product. 
+
+Use a header when:
+
+- you want links to top level pages to be persistent on screen
+- you expect users to switch between top level pages frequently
+
+![Primary Header template and example](images/header-generic.png)
+
+#### Header Responsive Behavior
+
+As a header scales down to fit smaller screen sizes, header links and menus should collapse into a left-nav hamburger menu. The left nav menu (previously L1 menu) is reserved for this scenario and should not be used as primary navigation with the header. If an additional navigation menu is needed, use an app switcher placed on the far right side of the header.
+
+![Responsive Header example](images/header-responsive.png)
 
-1. **Platform icon or L1 menu icon.** If your platform includes a L1 product-switcher (e.g. IBM Cloud), then this space will hold a hamburger menu to activate the switcher. If not, the platform icon sits here.
+#### Header Elements
 
+1. **Platform icon/Hamburger Menu** The platform icon appears in this space at wider browser widths, and switches to a hamburger menu to serve as overflow for header navigation links as the window narrows.
 - **Platform name** (e.g. Cloud, IoT, Watson). The platform name is always preceded by "IBM."
 - **Up to 5 optional horizontal platform-levl nav links and/or dropdown menus.** If using a dropdown, include the down-pointing chevron after the link label. Dropdowns open on click and are closed by either selecting an item in the menu, clicking outside the menu area, or clicking on the menu label.
```

Comment:
typo `platform-levl` = platform-lev**e**l

Reply:
Actually I think we should change it to "top level" because they could be platform or product level depending.